src/main/resources/init.sh

```diff
@@ -13,26 +13,34 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#
-
-ACTION=$1
-SERVICE="@serviceName@"
-SERVICE_CMD="service/bin/$SERVICE"
-PIDFILE="var/run/$SERVICE.pid"
-ARGS="@args@"
-
-# uses SERVICE_HOME when set, else, traverse up two directories respecting symlinks
-SERVICE_HOME=${SERVICE_HOME:-$(cd "$(dirname "$0")/../../" && pwd)}
-cd "$SERVICE_HOME"
-
-source service/bin/config.sh
 
 is_process_active() {
    local PID=$1
    ps $PID > /dev/null;
    echo $?
 }
 
+# Everything in this init.sh script relative to the base directory of an SLSv2 distribution
+pushd "`dirname \"$0\"`/../.." > /dev/null
+APP_HOME="`pwd -P`"
+
+# Select launcher binary for this OS
+case "`uname`" in
+  Linux*)
+    LAUNCHER_CMD=service/bin/javalauncher-linux-amd64
+    ;;
+  Darwin*)
+    LAUNCHER_CMD=service/bin/javalauncher-darwin-amd64
```

Comment:
- for testing when developing this plugin
- for local dev testing for users of the distribution pluginb